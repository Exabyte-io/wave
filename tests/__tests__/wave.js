import React from 'react';
import {shallow} from 'enzyme';
import renderer from 'react-test-renderer';

class Welcome extends React.Component {
    render() {
        return (
            <div>Hello world</div>
        );
    }
}

describe('Welcome', () => {
    it('renderer', () => {
        const component = renderer.create(<Welcome/>);
        const json = component.toJSON();
        expect(json).toMatchSnapshot();
    });

    it('enzyme', () => {
        const welcome = shallow(<Welcome/>);
        expect(welcome.find('div').text()).toEqual('Hello world');
    });
});
