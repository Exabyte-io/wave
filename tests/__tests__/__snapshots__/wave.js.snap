version https://git-lfs.github.com/spec/v1
oid sha256:b4d883d8a0302ea09c9eb0a1d2c7d64bc10f59f3b7f8a85c91ffd0d87471f56c
size 305
